Changelog.md

```diff
@@ -16,7 +16,7 @@
 
 ## 4.5.0
 
- * `{{t}}` helper supports a context object as a second ordered argument;
+ * `{{translate}}` helper supports a context object as a second ordered argument;
```

Comment:
These old changelog entries shouldn't change. They represent snapshots in time, not the current state.